moped-database/views/project_list_view.sql

```diff
@@ -95,6 +95,58 @@ parent_child_project_map AS (
     LEFT JOIN project_council_district_map project_and_children_districts ON projects.self_and_children_project_ids = project_and_children_districts.project_id
     LEFT JOIN project_council_district_map project_districts ON projects.project_id = project_districts.project_id
     GROUP BY projects.project_id
+),
+
+min_confirmed_phase_dates AS (
```

Comment:
🔖 from here down to 219 is new and updates the way the project list view get the `substantial_completion_date` and then uses it in a case statement to get `substantial_completion_date_estimated`